runbotics/runbotics-orchestrator-ui/src/src-app/views/credentials/CredentialsCollection/EditCredentialsCollection/SharedWithUsers/SharedWithUsers.tsx

```diff
@@ -49,6 +51,7 @@ export const SharedWithUsers: FC<SharedWithUsersProps> = ({ credentialsCollectio
         );
     }, [nonAdmins.all]);
 
+    // TODO - to be updated after users migration to include all users in tenant
```

Comment:
still waiting for that migration?

Reply:
yes and I think it will be needed to have separate task for it, cause it is used also e.g. in process collections